logservice.go

```diff
@@ -42,6 +42,8 @@ func parseFlags() app {
 	flag.StringVar(&a.buildID, "build", "", "ID of the build that is emitting logs ($SD_BUILDID)")
 	flag.StringVar(&a.token, "token", "", "JWT for authenticating with the Store API ($SD_TOKEN)")
 	flag.IntVar(&a.linesPerFile, "lines-per-file", defaultLinesPerFile, "Max number of lines per file when uploading ($SD_LINESPERFILE)")
+	flag.BoolVar(&a.isLocal, "local-mode", false, "Build run in local mode")
+	flag.StringVar(&a.artifactsLogFile, "artifacts-log-file", "", "Path to the log file in Artifacts directory in local mode")
```

Comment:
This argument require file path.  I think it is better to fix description.

Reply:
I fixed it.